src/app/clusters/boolean-state-server/boolean-state-cluster.h

```diff
@@ -31,7 +31,11 @@ class BooleanStateCluster : public DefaultServerCluster
                                                 AttributeValueEncoder & encoder) override;
     CHIP_ERROR Attributes(const ConcreteClusterPath & path, ReadOnlyBufferBuilder<DataModel::AttributeEntry> & builder) override;
 
-    void SetStateValue(bool stateValue, EventNumber * eventNumber = nullptr);
+    // Set a boolean value
+    // If the boolean value was actually modified, an event will be generated
+    // On success, the return value is an optional containing an EventNumber (alias for uint64_t)
```

Comment:
```suggestion
    // On success, the return value is an optional containing an EventNumber
```

I think alias is an implementation detail. You get an EventNumber whatever that is, we do not need to couple this comment with the definition of EventNumber